tests/turtle/syntax/manifest.ttl

```diff
@@ -3,15 +3,24 @@
 ## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
 ## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
 
-PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#>
-PREFIX test:   <http://www.w3.org/2001/sw/DataAccess/tests/>
-PREFIX rdft:   <http://www.w3.org/ns/rdftest#>
-PREFIX trs:    <https://w3c.github.io/rdf-star/tests/turtle/syntax#>
-
+PREFIX rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> 
+PREFIX rdfs:   <http://www.w3.org/2000/01/rdf-schema#> 
+PREFIX mf:     <http://www.w3.org/2001/sw/DataAccess/tests/test-manifest#> 
+PREFIX rdft:   <http://www.w3.org/ns/rdftest#> 
+PREFIX trs:    <https://w3c.github.io/rdf-star/tests#> 
```

Comment:
Alignment.

Reply:
@afs -- What alignment are you looking for?  This appears to me to have a reasonable alignment, though it's not exactly to my taste.  I think you'll be more likely to get your desired alignment if you specify it.

My own preference is to alpha-sort the prefixes (helps avoid accidental duplication), then left-align `PREFIX`, right-align the prefixes on their `:`, and left align the URLs.

Reply:
Whitespace.
In the PR, the tabs cause problems in GH interface and in some places there is extra spaces as well.